src/api/models/user.js

```diff
@@ -23,7 +22,11 @@ const userSchema = new Schema(
     birthday: Date,
     gender: Number,
     action: Boolean,
-    role: Number,
+    role: {
+      type: String,
+      enum: ['superAdmin', 'admin', 'member'],
```

Comment:
You should get role type from config file so that we can reuse in repository for more consistence